### DISCUSSION TITLE
Get image url from next/image

### DISCUSSION THREAD

**Discussion by dhvcc** [vercel/next.js#42885] [Ideas]
### Describe the feature you'd like to request

Would be nice if next/image would forward refs to `<img/>` or provide more data like the url in `onLoadingComplete`. I have a case where I load the same image twice:
1. As an actual image
2. As a blurred background

The thing is that background image uses CSS's `background-image: url("...")`. I'd like to reuse cached image that was loaded initially, but there's no way to get this data from event or even access the ref

### Describe the solution you'd like

Either forward the ref (I don't see an issue here + it would stop issues similar to this popping up from time to time) or add url to data given to `onLoadingComplete` callback

### Describe alternatives you've considered

For now I'm using a workaround like 
```js
`/_next/image?url=${encodeURIComponent(url)}&w=1080&q=75`
```
Which I'd really like to remove.

Also, to note, this is an edge case where I **can't** just use a regular url in CSS. I **have** to reuse the cached data or the image won't load.

**Comment by dhvcc** [comment]
An update to this issue - I also have a case where I need to use ref for `IntersectionObserver` and, sadly, `next\image` doesn't allow to use it without doing weird code

**Comment by jankaifer** [comment]
`onLoadingComplete` should give you `<img/>` reference since `next@13`.
docs: https://nextjs.org/docs/api-reference/next/image#onloadingcomplete

I'm working on `ref` here: https://github.com/vercel/next.js/pull/43193
